### PR TITLE
Feature/ranges

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What does this PR do? \*
Adds `product-list-price-range` and `product-selling-price-range`
Also, `product-list-price` and `product-list-price-range` are only rendered if they are different from the selling price.

#### How to test it? \*

[Workspace](https://productprice--storecomponents.myvtex.com/)

### CODE REVIEW

File: react/ListPriceRange.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          minPriceValue: (
+            <span className={`${handles.listPriceRangeMinValue} strike`}>
+              <FormattedCurrency value={minPrice} />
+            </span>
+          ),
+          maxPriceValue: (
+            <span className={`${handles.listPriceRangeMaxValue} strike`}>
+              <FormattedCurrency value={maxPrice} />
+            </span>
+          ),
```
Comment (klzns) [inline react/ListPriceRange.tsx]: What do you think in the case of not existing a range, only make the variable `listPriceValue` available?

File: react/SellingPriceRange.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          minPriceValue: (
+            <span className={handles.sellingPriceRangeMinValue}>
+              <FormattedCurrency value={minPrice} />
+            </span>
+          ),
+          maxPriceValue: (
+            <span className={handles.sellingPriceRangeMaxValue}>
+              <FormattedCurrency value={maxPrice} />
+            </span>
+          ),
```
Comment (klzns) [inline react/SellingPriceRange.tsx]: Same thing here, only make the variable `sellingPriceValue` available if is noRangeMessage

File: react/ListPriceRange.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  'listPriceRangeMaxValue',
+] as const
+
+const ListPriceRange: StorefrontFC<PriceRangeProps> = props => {
+  const { message, noRangeMessage, markers } = props
+  const handles = useCssHandles(CSS_HANDLES)
+  const { product } = useContext(ProductContext)
+
+  const priceRange = product?.priceRange
+  if (!priceRange) {
```
Comment (victorhmp) [inline react/ListPriceRange.tsx]: ⛏ [Nitpick] Why not add this conditional to the one below it? Would that `if` become too bloated?

File: react/SellingPriceRange.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+              }
+            : {
+                sellingPriceValue: (
+                  <span className={handles.sellingPriceRangeUniqueValue}>
+                    <FormattedCurrency value={maxPrice} />
+                  </span>
+                ),
+              }
+        }
+      />
```
Comment (klzns) [inline react/SellingPriceRange.tsx]:
```suggestion
hasRange ?
<IOMessageWithMarkers
  message={message}
  markers={markers}
  handleBase="sellingPriceRange"
  values={{
    minPriceValue: (
      <span className={handles.sellingPriceRangeMinValue}>
        <FormattedCurrency value={minPrice} />
      </span>
    ),
    maxPriceValue: (
      <span className={handles.sellingPriceRangeMaxValue}>
        <FormattedCurrency value={maxPrice} />
      </span>
    ),
  }}
/>
: <IOMessageWithMarkers
  message={noRangeMessage}
  markers={markers}
  handleBase="sellingPriceRange"
  values={{
    sellingPriceValue: (
      <span className={handles.sellingPriceRangeUniqueValue}>
        <FormattedCurrency value={maxPrice} />
      </span>
    ),
  }}
/>
```

**Pull request merged**